presidio-anonymizer/presidio_anonymizer/entities/analyzer_results.py

```diff
@@ -0,0 +1,59 @@
+"""List wrapper of AnalyzerResult which sort the list using AnalyzerResult.__gt__."""
+from typing import List
+
+from presidio_anonymizer.entities import AnalyzerResult
+
+
+class AnalyzerResults(list):
+    """
+    A class which provides operations over the analyzer result list..
+
+     It includes removal of unused results and sort by indices order.
+     Additional information about the rational of this class:
+    - One PII - uses a given or default transformation to anonymize and replace the PII
+    text entity.
+    - Full overlap of PIIs - When one text have several PIIs, the PII with the higher
+    score will be taken.
+    Between PIIs with similar scores, the selection will be arbitrary.
+    - One PII is contained in another - anonymizer will use the PII with larger text.
+    - Partial intersection - both will be returned concatenated.
+    """
+
+    def to_sorted_unique_results(self, reverse=False) -> List[AnalyzerResult]:
```

Comment:
I don't see why letting to reverse, and having false as default when the only usage is for reverse=True